tests/testthat/test-gist.R

```diff
@@ -0,0 +1,7 @@
+context("gists")
+
+test_that("animint2gist() returns an object of class 'gist'", {
+  g <- animint2gist(list(p = qplot(1:10)), screenshot = FALSE)
+  expect_is(g, "gist")
+  gistr::delete(g)
```

Comment:
Believe it! 

![excited_beyond_belief](https://cloud.githubusercontent.com/assets/1365941/6927069/7254387e-d7b0-11e4-9d99-c3111ca23f51.gif)